UnitTests/classifierUnitTest/classifierUnitTest.m

```diff
@@ -0,0 +1,118 @@
+% Test the classifier yields the correct accuracy
```

Comment:
Can you be more specific? I don't really understand what this is all testing

Reply:
I'll move this entire unit test to the gitbook. I don't think it meshes with how hctsa is set up at the moment